schema/config-schema.json

```diff
@@ -144,6 +144,10 @@
                 "selinuxLabel": {
                     "type": "string"
                 },
+                "ioPriority": {
+                    "class": "string",
+                    "priority": "integer"
+                },
```

Comment:
should it be an enumeration of the supported values?

Reply:
@giuseppe Thanks for your review. It would be better! I have fixed it.